std/math.d

```diff
@@ -4589,7 +4589,7 @@ unittest
 
 
 /*********************************
- * Returns !=0 if e is a NaN.
+ * Returns $(D_KEYWORD true) if x is a NaN, and $(D_KEYWORD false) otherwise. 
```

Comment:
also, saying `and false otherwise` is redundant, as there is no other possible value.